### PR TITLE
Guest registration

### PR DESCRIPTION
**Pull request by townsend3721**

### CODE REVIEW

File: app/assets/javascripts/guests.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (harperiscoding) [inline app/assets/javascripts/guests.js:1]: clear unused generated files

File: app/assets/stylesheets/guests.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the guests controller here.
```
Comment (harperiscoding) [inline app/assets/stylesheets/guests.scss:1]: clear unused generated files

File: app/views/guests/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Enter New Guest Information
+<hr>
+<%= form_for @guest do |f| -%>
+	FirstName: <%= f.text_field :firstname%><br />
+	Last Name: <%= f.text_field :lastname%><br />
+	Email: <%= f.text_field :email%><br />
+	Event: <%= f.text_field :event%><br />
```
Comment (harperiscoding) [inline app/views/guests/new.html.erb:7]: probably should be able to select from a list of events.

File: config/secrets.yml [hunk 4]
```diff
@@ -18,7 +18,7 @@
 # Environmental secrets are only available for that specific environment.
 
 development:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE_DEV"] %>
```
Comment (harperiscoding) [inline config/secrets.yml:21]: follow setup instructions.

File: test/controllers/guests_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GuestsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
```
Comment (harperiscoding) [inline test/controllers/guests_controller_test.rb:5]: we don't use test directory at all, delete unused

**Review by harperiscoding — CHANGES_REQUESTED** [review]
make rspec/cucumber tests, delete unused stuff you're generating.

File: app/assets/stylesheets/guests.scss (hunk 2 above)
Comment (townsend3721) [reply app/assets/stylesheets/guests.scss:1]: Won't we use this file later when we start adding styles?

File: test/controllers/guests_controller_test.rb (hunk 5 above)
Comment (townsend3721) [reply test/controllers/guests_controller_test.rb:5]: Deleted

**Pull request closed without merging**